packages/pixeloven-webpack/prettier.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is it worth moving these prettier rules to the root and extending them like you've done with tsconfigs?

Reply:
Yeah I'd like to do that just never got around to it.